README.md

```diff
@@ -0,0 +1,83 @@
+# Bookish
+> The Kotlin Android app to read books in Cloud(in theory) or locally
+> 
+> View the code [_here_](https://github.com/AKushch1337/Bookish/tree/master/app/src).
+> 
+> View the design doc [_here_](https://docs.google.com/document/d/1qnaJQZAas_YInUklZ8cCjC-hOtiTIzA8HYjtcKa0vN8/edit#)
+-------------------------------
+## General Information
+- The application is a book reader. 
+- Supported doc formats will be: PDF, EPUB(maybe).
+- Ambition to add cloud syncing and Google auth using Firebase 
+- Minimum supported Android Version is **7.0**
+
+
+
+## Technologies Used
+- IDE: Android Studio
+- Language: Kotlin
+- Other instruments:
+   - Jetpack Compose
+   - Navigation Compose
+   - Coil
+   - Firebase
+      - Google authorization
+      - Cloud
+   - [Bouquet](https://github.com/GRizzi91/bouquet) framework
+   - Tests(UI Tests)
+      - jUnit4
+      - Compose UI testing tools
+      - Espresso
+
+## Build the project yourself
+
+- Clone the project
+
+- Move into the project directory
+
+- Build the project using Gradle Wrapper
+ ```gradle
+./gradlew build
+``` 
+- Or build it using Android Studio(that way you will run the app also!)
```

Comment:
Extract information from the parenthesis to the Run section, or merge the Build and Run sections.